### ISSUE TITLE
org.tensorflow.distruntime

### ISSUE THREAD

**Issue by zw97** [Intel-bigdata/TensorFlowOnYARN#51]
When I compile TensorflowOnYARN-hadoop-2.7.3, the following error message appears. Would you like to ask if the problem is that the Tensorflow source installation is incorrect or TensorflowOnYARN-hadoop-2.7.3 has special requirements for the version of tensorflow? 
    Hope to get your reply.

[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.2:compile (default-compile) on project tensorflow-bridge: Compilation failure: Compilation failure:
[ERROR] /home/pengyuanyuan/TensorFlowOnYARN-hadoop-2.7.3/tensorflow-parent/tensorflow-bridge/src/main/java/org/hdl/tensorflow/bridge/TFServer.java:[16,34] 程序包org.tensorflow.distruntime不存在
[ERROR] /home/pengyuanyuan/TensorFlowOnYARN-hadoop-2.7.3/tensorflow-parent/tensorflow-bridge/src/main/java/org/hdl/tensorflow/bridge/TFServer.java:[17,32] 程序包org.tensorflow.framework不存在
[ERROR] /home/pengyuanyuan/TensorFlowOnYARN-hadoop-2.7.3/tensorflow-parent/tensorflow-bridge/src/main/java/org/hdl/tensorflow/bridge/TFServer.java:[33,10] 找不到符号
[ERROR] 符号:   类 ServerDef
[ERROR] 位置: 类 org.hdl.tensorflow.bridge.TFServer

**Comment by manuzhang** [comment]
Sorry but the project is not actively maintained anymore. You may look at TensorFlow on YARN support in Hadoop 3.x

**Comment by zw97** [comment]
https://github.com/Intel-bigdata/HDL
Thank you for your answer. Can TensorflowOnYarn.tar.gz under this link support Hadoop 3.1.0? 
Hope to get your reply

**Comment by manuzhang** [comment]
I don't think that project is maintained either. Try looking for a solution from Hadoop or Tensorflow community.

**Comment by superbobry** [comment]
@zw97 @manuzhang you might want to try [`tf-yarn`](https://github.com/criteo/tf-yarn/). It's a pure Python library integrating TensorFlow directly with YARN.
